### PR TITLE
Build fails on Windows during 'bower install' phase when frontend-maven-plugin is executed

### PR DESCRIPTION
**Pull request by 2686747**
#849: delegated bower's postinstall to maven

### CODE REVIEW

**Comment by karato** [comment]
@2686747 Many thanks for the PR, let me find a reviewer for it

**Comment by karato** [comment]
@mkordas do this review

**Comment by mkordas** [comment]
@2686747 I'm on it

File: netbout-web/pom.xml [hunk 1]
```diff
@@ -311,45 +311,80 @@
...
+                    </plugin>
+                    <plugin>
+                        <artifactId>maven-antrun-plugin</artifactId>
+                        <executions>
+                            <!-- Move javascript files loaded by bower -->
+                            <execution>
+                                <phase>process-resources</phase>
+                                <configuration>
+                                    <target>
+                                        <mkdir dir="target/classes/js/thridparty/" />
```
Comment (mkordas) [inline netbout-web/pom.xml]: @2686747 we don't need space before `/>`

File: netbout-web/pom.xml [hunk 2]
```diff
@@ -311,45 +311,80 @@
...
+                                        <move todir="target/classes/js/thridparty/">
+                                            <fileset dir="target/js-tmp/jquery-textcomplete/dist/">
+                                                <include name="jquery.textcomplete.min.*"/>
+                                            </fileset>
+                                        </move>
+                                        <delete  dir="target/js-tmp"/>
```
Comment (mkordas) [inline netbout-web/pom.xml]: @2686747 let's use single space here

**Comment by mkordas** [comment]
@2686747 looks good, just 2 minor ones

**Comment by 2686747** [comment]
@mkordas thanks, fixed

**Comment by mkordas** [comment]
@2686747 looks nice

**Comment by mkordas** [comment]
@rultor merge

**Comment by dmzaytsev** [comment]
@rultor merge

**Comment by karato** [comment]
@elenavolokhova please, let us know what do you think about this ticket, according to our [QA rules](http://at.teamed.io/qa.html)

**Comment by karato** [comment]
@rultor deploy now pls

**Comment by elenavolokhova** [comment]
@mkordas 
According to our [quality rules](http://www.teamed.io/qa.html):
 > The code reviewer found at least three problems in the code.
 
 Only two issues were found during review.
Please confirm that you will try to find at least three problems next time.

**Comment by mkordas** [comment]
@elenavolokhova I confirm

**Comment by elenavolokhova** [comment]
@karato Quality is acceptable here.

**Comment by karato** [comment]
> @karato Quality is acceptable here.

@elenavolokhova thanks a lot, next time everybody should try to make it better

**Comment by karato** [comment]
@mkordas added 10 mins to @elenavolokhova, for QA review, payment code is `79953801`. **22 mins** added to your account (payment number `56e19a9fe401062ecf000014`), many thanks for your contribution! 13 hours and 52 mins spent here.. extra minutes for review comments (c=7). +22 to your rating, your total score is [+8564](http://www.netbout.com/b/36080?open=rating)

**Pull request merged**